kordex/build.gradle.kts

```diff
@@ -69,6 +70,7 @@ dependencies {
 }
 
 tasks.withType<ShadowJar> {
+    setProperty("zip64", true)
```

Comment:
Are you sure this is the right approach? The error you're fixing by doing this wasn't present before your PR, so I'm concerned by the apparent number of additional files.

Reply:
it doesn't seem to fix it anyw, however I didn't touch the build logic for this, so no idea what causes this

Reply:
I'll ask a Gradle guru about it on the Quilt server

Reply:
This actually builds fine on my Windows machine - did you try building it locally?